otter/log/formatters.py

```diff
@@ -30,6 +30,61 @@
 NON_PEP3101_SYSTEMS = ('kazoo',)
 
 
+_fanout = None
+
+
+def add_to_fanout(observer):
+    """
+    :return: the global instance of :class:`FanoutObserver`
```

Comment:
Good point!  I was mostly trying to preserve the call chain somewhat in the formatters, but hadn't really considered the usage elsewhere.  Thanks! :)